brink.conf

```
@@ -0,0 +1,5 @@
+BASE_REQUIREMENTS='pip==20.2.4 chevah-brink==0.78.1 paver==1.2.4'
+PYTHON_CONFIGURATION='default@2.7.18.bd461724:sol10@2.7.8.bd461724'
```

This is ok with simple default...we don't plan to use it on solaris 10 :)

```suggestion
PYTHON_CONFIGURATION='default@2.7.18.bd461724'
```